src/Protocol/ATProtocol/Processor.php

```diff
@@ -425,6 +425,10 @@ public function getHeaderFromPost(stdClass $post, string $uri, int $uid, int $pr
 		}
 
 		$account = Contact::selectFirstAccountUser(['pid'], ['id' => $contact['id']]);
+		if (empty($account)) {
```

Comment:
There should always be a corresponding `account-user` entry for each contact entry. If that is not the case, there is some problem in the system. So we shouldn't solve this differently, for example in some daily job that searches for such problems. We have several places all over the system where we expect to have `account-user` and `contact` aligned.

Reply:
I don't think one precludes the other. No matter the reason, if we don't get an `account-user` entry here, we can't go further as it's depending on it. Unless you imply we should create the record right here?

Reply:
That's in fact a good idea. We can create a function `selectFirstAccountUserForId`. When this fails, the function then checks if the id is present in the contact table. And when this is the case, it is created in the `account-user` table as well.

Nonetheless we need some daily job that fixes this issue as well, since we rely on the fact that these two tables are synchronized.

Reply:
Sounds good, but it's outside of the scope of this fix.

Reply:
I created a PR for that: https://github.com/friendica/friendica/pull/14846